sigal/themes/colorbox/templates/index.html

```diff
@@ -120,7 +120,7 @@
               <div style='display:none'>
                 <div id="{{ media.filename|replace('.', '')|replace(' ', '') }}">
                   <video controls>
-                  <source src='{{ media.filename }}' type='video/webm' />
+                  <source src='{{ media.filename }}' {% if not settings.use_orig %}type='video/webm'{% endif %}/>
```

Comment:
Nothing :)

I mean, I've been testing in Firefox and Epiphany, and didn't perceive anything weird: videos were played normally, without any kind of warning.